### ISSUE TITLE
Alternatives?

### ISSUE THREAD

**Issue by p3v9d5ui** [FranckFreiburger/vue-pdf#360]
This project appears to be dead -- there has been no maintenance in many months, and there are no responses to issues. Can anyone recommend a good alternative?

**Comment by dolbex** [comment]
You've probably found something by now @p3v9d5ui but I migrated over to https://github.com/hrynko/vue-pdf-embed#usage with success.

**Comment by p3v9d5ui** [comment]
Thanks @dolbex ... Looks promising, but seems to be very basic at the moment.

**Comment by dolbex** [comment]
Yeah, might be worth it to just wrap something around Mozilla's lib and roll your own if you're wanting to do something a little more advanced.

**Comment by dvago** [comment]
I believe just closed source projects such as pspdfkit, I'm looking at the codebase of this project to see how much time it'll take to create something meaningful for my case with pdf.js. 

It's a shame that the maintainer disappeared, he is definitely not dead as the last commit was on the 23rd of Jan in a different project

**Comment by vuepdfviewer** [comment]
Hey everyone,

I know it’s been a while since the last comment here, but I wanted to chime in with an alternative in case anyone’s still looking! If you’re searching for a modern and actively maintained PDF viewer for Vue, you might want to check out [Vue PDF Viewer](https://www.vue-pdf-viewer.dev/?utm_source=github&utm_medium=issues&utm_content=FranckFreiburger-vue-pdf-360).

It’s a Vue 3-based library with built-in features like search, zoom controls, and customizable toolbars—plus, it leverages PDF.js under the hood for reliable rendering. I’m one of the core member, so if you have any questions, feel free to reach out.

Hope this helps!
